docs/running from source.md

```diff
@@ -7,7 +7,7 @@ use that version. These steps are for developers or platforms without compiled r
 ## General
 
 What you'll need:
- * [Python 3.10.11 or newer](https://www.python.org/downloads/), not the Windows Store version
+ * [Python 3.11.13 or newer](https://www.python.org/downloads/), not the Windows Store version
```

Comment:
3.11.9 appears to be the last one to have a Windows download.
So while you should not host a server with it, for local testing it should be fine.

```suggestion
 * [Python 3.11.9 or newer](https://www.python.org/downloads/), not the Windows Store version
```